src/Presentation/Console/Command/ShowWorkflowCommand.php

```diff
@@ -17,12 +17,15 @@
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\BufferedOutput;
 use Symfony\Component\Console\Output\OutputInterface;
-use Symfony\Component\Console\Style\SymfonyStyle;
 use Symfony\Component\Process\Process;
-use Throwable;
 
 class ShowWorkflowCommand extends Command
 {
+    /**
+     * @var string
+     */
+    protected const WORKFLOW_VAR_DIR = 'var/workflow';
```

Comment:
you can move it to service param. and concatenate absolute path there

Reply:
Also const naming should be in Yoda style => `VAR_DIR_WORKFLOW`